packages/api-graphql/src/types/index.ts

```diff
@@ -28,7 +28,7 @@ export enum GRAPHQL_AUTH_MODE {
 
 export interface GraphQLResult<T = object> {
 	data?: T;
-	errors?: [object];
+	errors?: Array<{ [key: string]: any; }>;
```

Comment:
Thanks for the update @neilime! How do you feel about using `GraphQLError` from `graphql` so we can be more specific about what `errors` is:
https://github.com/aws-amplify/amplify-js/blob/40c4e21b8cab18dd203e942083d8b6e16ed06997/packages/api-graphql/src/GraphQLAPI.ts#L291

Maybe something like this:
```suggestion
	errors?: GraphQLError[];
```

and you might need to import it like this:
```js
import { GraphQLError } from 'graphql/error/GraphQLError';
```

I haven't tested this locally, but let me know what you find!

Reply:
@amhinson using GraphQLError is better and it works !